### ISSUE TITLE
Icon inside a .collection-item not aligned properly

### ISSUE THREAD

**Issue by cnicodeme** [Dogfalo/materialize#4241]
### Description
Hi,

When I add an icon inside a .collection-item, the text for the collection-item is not aligned with the icon.

### Repro Steps
http://codepen.io/anon/pen/GreEaP

The left icon should be aligned with the text. Which is not.

Thanks :)

**Comment by marcosvicente** [comment]
<span class="test">Localhost</spam>

.test{
	vertical-align: top;
}

**Comment by cnicodeme** [comment]
The class `test` doesn't exist. What are you trying to do?

**Comment by marcosvicente** [comment]
<span class="test">Localhost</span>

.test{
vertical-align: top;
}

**Comment by marcosvicente** [comment]
"<span class="test">localhost</span>"

**Comment by marcosvicente** [comment]
`<span class="test">localhost</span>`

**Comment by marcosvicente** [comment]
Sorry

**Comment by cnicodeme** [comment]
Yes ok, thank you.

Something like this is better too : 

```
.test {
    vertical-align: top;
    padding-top: 2px;
    display: inline-block
}
```

**Comment by tomscholz** [comment]
Just add the `.left`class to the icon.
https://codepen.io/tomscholz/pen/mRogQN

**Comment by cnicodeme** [comment]
Yes indeed, thanks for the help!

**Comment by tomscholz** [comment]
@cnicodeme I would also recommend to remove the span from the delete icon. I updated the pen here: https://codepen.io/tomscholz/pen/mRogQN

**Comment by cnicodeme** [comment]
I was wondering why the changes you both suggested didn't worked on my code.

I now know why. The collection have issues when used inside a sidenav item.

Here's the updated code that shows the (previously working) example : 
https://codepen.io/anon/pen/VPNKrE

**Comment by tomscholz** [comment]
2 questions:
1. Why do you need a collection inside your sidenav?
2. Do you work with sass?

**Comment by tomscholz** [comment]
https://codepen.io/anon/pen/VPNKrE

**Comment by cnicodeme** [comment]
1. Because the sidenav will contain a list of links to use (and the possibility to delete them). It's kind of like a menu, but that can be hidden or displayed.
2. I use scss, why?

**Comment by acburst** [comment]
I've made some sidenav styling a little more specific so this should work

Fixed in 28dce92
